src/search.ts

```diff
@@ -0,0 +1,10 @@
+export function searchIgnoreCase(
```

Comment:
```suggestion
export function includesIgnoreCase(
```
I would suggest naming this function after the function it originates from.